ivre/db/mongo.py

```diff
@@ -919,6 +919,14 @@ class MongoDBActive(MongoDB, DBActive):
                 ],
                 {"sparse": True},
             ),
+            (
+                [("ports.scripts.ntlm-info.NetBIOS_Domain", pymongo.ASCENDING)],
+                {"sparse": True},
+            ),
+            (
+                [("ports.scripts.ntlm-info.Product_Version", pymongo.ASCENDING)],
+                {"sparse": True},
+            ),
```

Comment:
I think this should be added to the `schema_migrations_indexes` attribute, so that they get created when a user migrates its data. What do you think?